wdl/src/main/scala/wdl/expression/WdlStandardLibraryFunctions.scala

```diff
@@ -394,6 +420,12 @@ class WdlStandardLibraryFunctionsType extends WdlFunctions[WomType] {
       val badArgs = params.mkString(", ")
       Failure(new Exception(s"Unexpected arguments to function `length`. `length` takes a parameter of type Array but got: $badArgs"))
   }
+  def flatten(params: Seq[Try[WomType]]): Try[WomType] = {
+    params.toList match {
+      case Success(WomArrayType(WomArrayType(t))) :: Nil => Success(WomArrayType(t))
```

Comment:
I suggest you use `extractSingleArgument` to be like the other one-argument functions

Reply:
Still todo?